.buildkite/bk.integration.pipeline.yml

```diff
@@ -345,12 +345,19 @@ steps:
             - "basic,slim,complete,service,elastic-otel-collector"
             - "wolfi,slim-wolfi,complete-wolfi,elastic-otel-collector-wolfi"
             version:
-            - v1.27.16
-            - v1.28.9
-            - v1.29.8
-            - v1.30.8
-            - v1.31.0
-            - v1.32.0
+            - v1.25.16 # [1]
```

Comment:
actually is this still true for v1.25.16? the link says EoL 1st of May 2025 🙂 maybe we can remove it?

Reply:
Good catch, removed in 7343c07fdb.  Thanks.